lib/kafo/kafo_configure.rb

```diff
@@ -348,9 +356,7 @@ def set_options
       end
 
       modules.each do |mod|
-        app_option d("--[no-]enable-#{mod.name}"),
-                   :flag,
-                   "Enable '#{mod.name}' puppet module",
+        app_option d("--[no-]enable-#{mod.name}"), :flag, "Enable '#{mod.name}' puppet module",
```

Comment:
I'm always a fan of avoiding formatting changes in the same change. That makes it harder for a reviewer to see what's really changed.